src/retrace/retrace.py

```diff
@@ -2122,8 +2124,25 @@ def strip_extra_pages(self) -> None:
             log_warn("makedumpfile exited with %d" % child.returncode)
             if newvmcore.is_file():
                 newvmcore.unlink()
+            return 
+
+        #Calculate the % of storage space saved with the new makedumpfile core created. 
+        elif ((oldsize - newvmcore.stat().st_size) / int(oldsize) * 100 < CONFIG["VmcoreDumpSavePercent"]):
+            log_warn("makedumpfile did not strip enough pages (needed to save %{}). 
+                    Not saving new vmcore.".format(CONFIG["VmcoreDumpSavePercent"]))
```

Comment:
String literals cannot be split over lines like this, but you can concatenate them simply by juxtaposition:
```suggestion
            log_warn("makedumpfile did not strip enough pages (needed to save "
                     "at least %{}). Not saving new vmcore.".format(CONFIG["VmcoreDumpSavePercent"]))
```